src/OpenColorIO/Config.cpp

```diff
@@ -249,11 +249,11 @@ static constexpr unsigned LastSupportedMinorVersion = OCIO_VERSION_MINOR;
 class Config::Impl
 {
 public:
-    enum Sanity
+    enum Validation
     {
-        SANITY_UNKNOWN = 0,
-        SANITY_SANE,
-        SANITY_INSANE
+        VALIDATION_UNKNOWN = 0,
+        VALIDATION_VALID,
+        VALIDATION_INVALID
```

Comment:
Possible alternatives that read slightly better:

- CONFIG_UNTESTED
- CONFIG_VALID
- CONFIG_INVALID

Alternative naming, avoiding the term "invalid":

- VALIDATION_UNKNOWN (maybe also UNTESTED or UNCHECKED)
- VALIDATION_PASSED
- VALIDATION_FAILED

Reply:
Good idea, I prefer PASSED/FAILED.

Reply:
That's better enum names.

Reply:
Renamed in https://github.com/AcademySoftwareFoundation/OpenColorIO/pull/1081/commits/6c1cceaa1992c4523b25f19e43cda692fa7235b6